frontend/src/metabase/sharing/components/AddEditSidebar.jsx

```diff
@@ -363,3 +398,55 @@ ChannelFields.propTypes = {
   channelSpec: PropTypes.object.isRequired,
   onChannelPropertyChange: PropTypes.func.isRequired,
 };
+
+function DefaultParametersSection({
+  className,
+  parameters,
+  defaultParametersById,
+}) {
+  const formattedParameterValues = formatDefaultParamValues(
+    parameters,
+    defaultParametersById,
+  );
+
+  return (
+    <div className={cx(className, "text-bold")}>
+      <Heading>
+        {t`Filter values`}
+        <Icon
+          name="info"
+          className="text-medium ml1"
+          size={12}
+          tooltip={t`You can customize filter values for each subscription with the Enterprise edition.`}
+        />
+      </Heading>
+      <div className="pt1 text-small text-normal text-medium">{t`If a dashboard filter has a default value, it’ll be applied when your subscription is sent.`}</div>
+      {formattedParameterValues.map((formattedValue, index) => {
+        return (
+          <div className="pt1 text-medium" key={index}>
+            {formattedValue}
+          </div>
+        );
+      })}
+    </div>
+  );
+}
+
+DefaultParametersSection.propTypes = {
+  className: PropTypes.string,
+  parameters: PropTypes.array.isRequired,
+  defaultParametersById: PropTypes.object.isRequired,
+};
+
+// TODO: will need improved formatting for operator parameter filters
+function formatDefaultParamValues(parameters, defaultParametersById) {
+  return parameters
+    .map(parameter => {
+      const value = conjunct(
+        [].concat(defaultParametersById[parameter.id]),
+        t`and`,
```

Comment:
"or" _might_ make more sense for multiple values. Though, I guess if it's describing the values in the filter rather than what the rows are filtered with "and" is better.